crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -64,6 +64,24 @@ def services
     end
   end
 
+  api :POST, "/api/upgrade/controllers", "Orchestrate the upgrade of controller nodes"
```

Comment:
Why can't we choose which controller should be upgraded next? Both makes actually no real sense to me... I would expect if I call `/controllers` that all controllers get upgraded automatically or if I call `/controller` that I can choose the controller. Maybe it's not possible to upgrade all controllers but we can provide a selection via a get route. This would make it a little bit more dynamic. Also if we don't expose it via the UI a admin can still do it manual which some customers want to do.